src/cli/repl.rs

```diff
@@ -592,16 +575,6 @@ impl Repl<F> {
                 let (first_io, ..) = self.eval_expr(first)?;
                 self.env = first_io.expr;
             }
-            "set-limit" => {
-                let limit = self.peek_usize(cmd, args)?;
-                validate_non_zero("limit", limit)?;
-                self.limit = limit;
-            }
-            "set-rc" => {
-                let rc = self.peek_usize(cmd, args)?;
-                validate_non_zero("rc", rc)?;
-                self.rc = rc;
-            }
```

Comment:
These are not meta Lurk. But the feature should be added back in later when we have changeable REPL configs